cpu/nrf52/ldscripts/nrf52832xxaa.ld

```diff
@@ -18,10 +18,23 @@
  * @}
  */
 
+rom_length = 512K;
+
+INCLUDE multislot.ld
+
 MEMORY
 {
-    rom (rx)    : ORIGIN = 0x00000000, LENGTH = 512K
-    ram (rwx)   : ORIGIN = 0x20000000, LENGTH = 64K
+    rom (rx)    : ORIGIN = 0x00000000 + boot_offset, LENGTH = rom_length
+    ram (rwx)   : ORIGIN = 0x20000000,               LENGTH = 64K
+}
+
+SECTIONS
+{
+    /* Image headers for MCUBoot */
+    .imghdr (NOLOAD):
+    {
+      . = . + img_hdr;
+    } > rom
```

Comment:
I think keeping this variable in the linker script doesn't harm, and adds readability when looking at the ELF file where we can find the specific allocated space for the headers. This can be useful when using different tools to add these headers directly to the ELF file, instead if padding them to the binary file. It also helps to know while debugging where the real image starts, while also knowing why the vector table is located 512B after the bootloader.